pytensor/tensor/basic.py

```diff
@@ -3726,109 +3727,18 @@ def trace(a, offset=0, axis1=0, axis2=1):
     return diagonal(a, offset=offset, axis1=axis1, axis2=axis2).sum(-1)
 
 
-class AllocDiag(Op):
-    """An `Op` that copies a vector to the diagonal of a zero-ed matrix."""
+class AllocDiag(OpFromGraph):
+    """
+    Wrapper Op for alloc_diag graphs
+    """
 
-    __props__ = ("offset", "axis1", "axis2")
+    __props__ = ("axis1", "axis2", "is_inline")
```

Comment:
So I think some of tour confusing is what happens when you inline. When you inline the OFG ceases to be, it's replaced by it's inner graph. As such you couldn't ever find an OFG with `is_inline=True` (assuming this is something you wanted to set to True when you inline it)?

```suggestion
    __props__ = ("axis1", "axis2")
```

Reply:
Yeah I get that now. Was definitely confused on this point before.